### ISSUE TITLE
change backbone

### ISSUE THREAD

**Issue by mehrdadazizi72** [biubug6/Pytorch_Retinaface#8]
@biubug6 hi, thanks for sharing your project.
can you provide short wiki for changing backbone?
how can I change the backbone and train?

**Comment by xsacha** [comment]
It's quite easy!
See in retinaface.py where the backbone is set. Change that to a different torch model.
Example: 

```
    elif net == 'detnas':
            self.backbone = ShuffleNetV2DetNAS(model_size='VOC_RetinaNet_300M')
            checkpoint = torch.load("VOC_RetinaNet_300M.pkl", map_location=torch.device('cpu'))
            self.backbone.load_state_dict(checkpoint)
            return_layers = {'6': 1, '9': 2, '16': 3}
```

The return layers just need to be consecutive layers from the model where the size doubles (eg. 32 -> 64 -> 128).
Then set the in_channels_stage2 to half of the first layer's channels.

**Comment by mehrdadazizi72** [comment]
@xsacha  Hi, I want to use efficientNet as a backbone but I'm confused about what should I do.
the efficinetnet repo : https://github.com/lukemelas/EfficientNet-PyTorch
there is any point?

**Comment by xsacha** [comment]
I tried this but there is a big issue with all the efficient net backbones I attempted. One of the operations wasn't very efficient in PyTorch (swish?). It used a lot of memory and wasn't any faster.
The one you linked was the one I tested.

**Comment by mehrdadazizi72** [comment]
@xsacha thanks for the quick reply.
for some reason, I should try this and meet some issues.
can you guide me to train by this?

**Comment by quocnhat** [comment]
Hi @xsacha ,
For Mobilenet v3, the sizes of the consecutive layer are 16, 24, 40, 48, ... which are not the double size of the last layer. So how can we return layers for that? 
If I modified the net, It works but low accuracy.
